### DISCUSSION TITLE
Does the new version of LGTVCompanion work with the older webOS?

### DISCUSSION THREAD

**Discussion by alimbada** [JPersson77/LGTVCompanion#220] [General]
As per title, does the new version of LGTVCompanion (4.x) still work with the older webOS version? I'm currently on webOS 04.40.93

**Comment by JPersson77** [comment]
Hi, yes! There should be no difference compared to earlier versions of the app. You may need to switch to legacy connection method for the device.

Are you experiencing issues?

**Reply by alimbada** [reply]
No issues. Just checking before I upgrade.
